bundles/binding/org.openhab.binding.zwave/database/products.xml

```diff
@@ -1705,6 +1705,15 @@
             <Label lang="en">Roller Shutter</Label>
             <ConfigFile>fibaro/fgrm222.xml</ConfigFile>
         </Product>
+		<Product>
```

Comment:
Ah, I see. You guys use spaces rather than tabs, which I have used, for indentation. How old-fashioned :)

<img width="423" alt="screen shot 2017-03-23 at 14 59 48" src="https://cloud.githubusercontent.com/assets/22912705/24254920/00b340f0-0fdc-11e7-933c-943db31b455e.png">